main.go

```diff
@@ -152,6 +154,8 @@ func getPrinter() output.Printer {
 	fr := new(LocalFileReader)
 	if *html {
 		return output.NewHtmlPrinter(os.Stdout, fr)
+	} else if *plumbing {
+		return output.NewPlumbingPrinter(os.Stdout, fr)
```

Comment:
Minor, but I think -plumbing flag should take higher precedence over -http if you happen to supply them both (even better, that should be an error).